sites/fast-website/src/app/components/feature-card/feature-card.template.ts

```diff
@@ -0,0 +1,14 @@
+import { html } from "@microsoft/fast-element";
+import { FeatureCard } from "./feature-card";
+
+export const FeatureCardTemplate = html<FeatureCard>`
+    <header>
+        <slot name="header"></slot>
```

Comment:
I think the default slot should be what you have as header currently. I don't think volume of content is important here. I tend to think in terms of long term extensibility as the design changes the constant for something like this is going to be some kind of "heading" or "header" area which represents whatever is dropped into the content area.

Reply:
OK, I will make that change.

Reply:
I've switched things up here.  Default slot goes to the first div with .card_header class and the second div, with .card_body class, has a slot for body content and a footer slot for the anchors.